x/shared/session.go

```diff
@@ -76,7 +77,7 @@ func GetClaimWindowOpenHeight(sharedParams *sharedtypes.Params, queryHeight int6
 
 	// An additional block is added to permit to relays arriving at the last block
 	// of the session to be included in the claim before the smt is closed.
-	sessionGracePeriodEndHeight := GetSessionGracePeriodEndHeight(sessionEndHeight)
+	sessionGracePeriodEndHeight := GetSessionGracePeriodEndHeight(sharedParams, sessionEndHeight)
 	claimWindowOpenOffsetBlocks := int64(sharedParams.GetClaimWindowOpenOffsetBlocks())
 	return claimWindowOpenOffsetBlocks + sessionGracePeriodEndHeight + 1
```

Comment:
Do you know why we have a +1 here?

Related to the comments I left here: https://github.com/pokt-network/poktroll/pull/571

Reply:
See https://github.com/pokt-network/poktroll/pull/571#discussion_r1630958871